packages/flutter/test/material/date_picker_test.dart

```diff
@@ -4,7 +4,9 @@
 
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
-import 'package:intl/intl.dart';
+import 'package:intl/intl.dart' hide TextDirection;
```

Comment:
as intl

Reply:
Actually, removed this import altogether. It was used only in one place, which can use `MaterialLocalizations` for this purpose.